vermouth/molecule.py

```diff
@@ -674,6 +673,38 @@ def edges_between(self, n_bunch1, n_bunch2, data=False):
                 else:
                     yield (node1, node2, self.edges[node1, node2])
 
+    def _remove_interactions_with_node(self, node):
+        """
+        We iterate through the different interactions we have and
+        remove the interactions where the atoms to be deleted are present.
+        Further we also delete the entire interaction_type if it is
+        empty after all the necessary interactions have been deleted.
+        """
+        for name, interactions in self.interactions.items():
+            for interaction in interactions:
+                if node in interaction.atoms:
+                    self.interactions[name].remove(interaction)
+
+        for i in self.interactions.copy():
+            if not self.interactions[i]:
+                self.interactions.pop(i)
+
+    def remove_node(self, node):
+        '''Overriding the remove_node method of networkx
+           as we have to delete the interaction from the interactions list
+           separately which is not a part of the graph and hence does not
+           get deleted.'''
```

Comment:
Put the quotes on their own lines, just as with `_remove_interactions_with_node`